.travis.yml

```diff
@@ -1,34 +1,33 @@
-# See https://github.com/silverstripe/silverstripe-travis-support for setup details
 language: php
 
-sudo: false
-
 env:
   global:
-    - COMPOSER_ROOT_VERSION=1.0.x-dev
-    - CORE_RELEASE=4
-    # The path of the module when installed by composer
-    - MODULE_PATH=iframe
+    - COMPOSER_ROOT_VERSION=5.0.x-dev
 
 matrix:
-  fast_finish: true
   include:
     - php: 5.6
-      env: DB=MYSQL
+      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.0
-      env: DB=MYSQL
+      env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
 
 before_script:
-  - composer self-update || true
+  # Init PHP
   - phpenv rehash
   - phpenv config-rm xdebug.ini
+  - echo 'memory_limit = 2048M' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
+  # Install composer dependencies
   - composer validate
-  - git clone git://github.com/silverstripe/silverstripe-travis-support.git ~/travis-support
-  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
-  - cd ~/builds/ss
-  - composer install --prefer-dist
+  - composer require --no-update silverstripe/recipe-cms:1.0.x-dev silverstripe-themes/simple:~3.2
```

Comment:
Do you need the theme? If so we should probably use the installer rather than the CMS recipe, but if not just take it out